k9mail/src/main/java/com/fsck/k9/Account.java

```diff
@@ -320,6 +321,7 @@ protected Account(Context context) {
         mCryptoApp = NO_OPENPGP_PROVIDER;
         mCryptoKey = NO_OPENPGP_KEY;
         mCryptoSupportSignOnly = false;
+        mCryptoSupportDefaultOpportunistic = true;
```

Comment:
"support" is not the right word here, since even with this false the mode will still be "supported", just not enabled by default. Maybe `mCryptoDefaultDisabled`?